chroma-manager/tests/framework/unit/jenkins_steps/main

```diff
@@ -27,48 +27,6 @@ cd $CHROMA_DIR/chroma-manager
 make requirements
 pip install -r requirements.txt
 
```

Comment:
I'll add it. This was patch could not be applied and was erroring out. We probably stopped needing it a while ago.